graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/migration/CreateIndexQuery.java

```diff
@@ -52,7 +52,7 @@ public Table getTable() {
   @Override
   public String getDescription() {
     return String.format(
-        "Create %s index on %s.%s", index.name(), table.name(), index.column().name());
+        "Create index %s on %s.%s", index.name(), table.name(), index.column().name());
```

Comment:
Only tangentially related, but I noticed it while testing. It's all going to be squashed when we merge the `graphql-first` PR anyway.